### ISSUE TITLE
Smart Device V2 fan status issue

### ISSUE THREAD

**Issue by CaseySJ** [jonasmalacofilho/liquidctl#71]
Hello @jonasmalacofilho 

Line **451** of the _NZXT Smart Device V2_ driver is currently this:
```Python
        self._read_until({b'\x67\x04': parse_fan_info})
```
But this is incorrect. It results in:
```
NZXT Smart Device V2 (experimental)
├── Fan 1 duty         0  %
├── Fan 1 speed    12254  rpm
├── Fan 2 duty         0  %
├── Fan 2 speed    12150  rpm
├── Fan 3 duty        71  %
├── Fan 3 speed    12237  rpm
└── Noise level        0  dB
```
It should be changed to:
```Python
        self._read_until({b'\x67\x02': parse_fan_info})
```
Which results in the correct status:
```
NZXT Smart Device V2 (experimental)
├── Fan 2 duty      20  %
├── Fan 2 speed    626  rpm
└── Noise level     50  dB
```

**Comment by jonasmalacofilho** [comment]
Thanks for reporting this.

Already fixed it in the master branch.  Will do a release later today.

**Comment by jonasmalacofilho** [comment]
@CaseySJ 

I just released 1.3.2 with this fix.  Can you update our custom tap, as well as the under-review formula for homebrew-core?

**Comment by CaseySJ** [comment]
Certainly, will do so this evening. Thanks for the quick turnaround on this!!
